docs/API_DESIGN.md

```diff
@@ -628,5 +629,16 @@ CLASS Exchange
 ## `get_exchange_ids()`
 
 ```pseudocode!
-FUNCTION get_exchange_ids(pfi_did_uri: string, bearer_did: BearerDid): []string
+FUNCTION get_exchange_ids(
+  pfi_did_uri: string, 
```

Comment:
Non blocking but is it worth having a DID type?

Reply:
Yeah, it's an idea worthy of consideration. I default to primitive types (strings) where possible, because I like to minimize cross-dependency usage (a DID type would originate, and in fact already exists, inside of `web5-rs`), but that's my bias.